src/components/DpTreeList/DpTreeList.vue

```diff
@@ -100,6 +99,21 @@ export default {
     DpTreeListToggle
   },
 
+  provide () {
```

Comment:
As we do not use  `v-if`, this can be removed as well, i guess.

Reply:
done here https://github.com/demos-europe/demosplan-ui/pull/774/commits/6902a08cec6ba1322d2012e312ac89114ec7ab5b